src/app/tests/suites/certification/Test_TC_DIAG_TH_NW_1_2.yaml

```diff
@@ -28,36 +28,30 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
-    #disabled due to this issue #18244
     - label: "read configured Channel attribute value"
-      disabled: true
       command: "readAttribute"
       attribute: "channel"
       response:
-          value: 0
+          value: null
 
     - label: "Validate constraints of attribute: Channel"
       command: "readAttribute"
       attribute: "channel"
       response:
           constraints:
-              type: uint16
+              type: null
```

Comment:
This does not make sense.  The type is uint16 (but nullable).  `null` is not a type.  Please undo this change.